packages/salesforcedx-vscode-apex-replay-debugger/test/unit/activation/getDialogStartingPath.test.ts

```diff
@@ -0,0 +1,106 @@
+/*
+ * Copyright (c) 2022, salesforce.com, inc.
+ * All rights reserved.
+ * Licensed under the BSD 3-Clause license.
+ * For full license text, see LICENSE.txt file in the repo root or https://opensource.org/licenses/BSD-3-Clause
+ */
+
+import { LAST_OPENED_LOG_FOLDER_KEY } from '@salesforce/salesforcedx-apex-replay-debugger/src';
```

Comment:
This file should be in the `test/jest` directory vs `test/unit`. 
`test/unit` -> old school
`test/jest` -> new school